SOURCES/sympa-systemd-README.RPM.md

```diff
@@ -13,7 +13,107 @@ package.  A bunch of work is needed to start your Sympa service.
     Sympa and updates site configuration files.  In this process you must
     choose e-mail(s) of listmaster, database settings and so on.
 
-2 Setup database
```

Comment:
Configuration can refer database.  So I suppose database setup would be better to be done at first.

Reply:
My point is if one follows the steps in README.RPM closely and setup the database before setting up the DB, he will not get the mail to sympa@domain.tld sent by 'sympa.pl --health_check', thus the MTA needs to be setup before the DB.